lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -86,6 +86,10 @@ def file_nonzero?(filename)
         File.exist?(filename) && !File.zero?(filename)
       end
 
+      def list_files_in_directory(directory_path)
+        Dir.entries(directory_path).select { |f| !File.directory? f }
```

Comment:
Use cases might vary after getting the entries using directory_path so it would be good to move this logic under check itself.

Reply:
@kgaikwad I actually need the same method in some other PR. I guess method name `list_files_in_directory` itself make sense, i.e. only files not directories. 
I can move this under check but then I will need to define the same method in other PR it will be code repetition.
Let me know your thoughts on this.